hr_employee_age/models/hr_employee.py

```diff
@@ -17,10 +17,9 @@
 #    along with this program. If not, see <http://www.gnu.org/licenses/>.
 #
 ###############################################################################
-from datetime import datetime
 
-from openerp import fields, models, api
-from openerp.tools import DEFAULT_SERVER_DATE_FORMAT as OE_DFORMAT
+from openerp import api, fields, models
```

Comment:
Use `from odoo` instead of `from openerp`, as now in version 10 is the default namespace.